src/components/AvatarGroup/AvatarGroup.st.css

```diff
@@ -142,18 +172,21 @@
     align-items: center;
 }
 
-:size(large) .textButtonContainer,
-:size(medium) .textButtonContainer,
-:size(small) .textButtonContainer {
-    margin-left: 12px;
+:size(large) .textButtonContainer::before,
+:size(medium) .textButtonContainer::before,
+:size(small) .textButtonContainer::before {
+    content: '\00a0';
+    width: 12px;
 }
 
-:size(xSmall) .textButtonContainer {
-    margin-left: 10px;
+:size(xSmall) .textButtonContainer::before {
+    content: '\00a0';
+    width: 10px;
 }
 
-:size(xxSmall) .textButtonContainer {
-    margin-left: 8px;
+:size(xxSmall) .textButtonContainer::before {
+    content: '\00a0';
+    width: 8px;
```

Comment:
I think only this part is required.
The rest of the changes (above) aren't needed.
Since the  padding is canceled by the margins.

Reply:
No, the other changes are also needed, or the label does not position itself correctly (since margin is on the wrong side of the last avatar).
So the avatars happen to look correct in rtl, but their bounding box is a bit off.

Reply:
I think you just need to take into account the negative margins.
So for instance something like this:
```css
:vars {
  largeMargin: 14px;
}

.root:size(large) .avatarContainer {
    margin-left: calc(-1 * value(largeMargin));
}

.root:size(large) .textButtonContainer::before {
  content: '\00a0';
  width: calc(12px + value(largeMargin));
}
```
I tried this locally, I think this works.

Reply:
Not really, since margin is only there on one side, and it needs to be compensated for only in either ltr or rtl. So then the padding ends up different sizes.
Basically, without some sort of specific rtl handling we are not allowed to write "x-left" on anything

I think it's getting a bit hacky at this point - I feel like we are trying to run away from proper rtl handling. Either generated css, where all the left/right stuff is switched for [dir=rtl], or just being able to use [dir=rtl] in the css? Can we in stylable? It seems to work, but Stylable gets mad at me.

![Screen Shot 2019-10-02 at 4 05 48 PM](https://user-images.githubusercontent.com/22763470/66046512-973b1a00-e52e-11e9-908a-6a4217b3e140.png)

Reply:
But 'ltr' is working for the avatargroup (without the text), so i'm not so sure why the change to `.avatarContainer` css rules is necessary?
Sorry for nagging, I just really prefer not adding the `rtl` flag.
Regarding `[dir=rtl]`, it's better to use the `rtl` state than using this selector, if we decide to use the `rtl` state.
I'm just not sure yet that it's needed.

Reply:
Maybe I'll add extra container to contain the Avatars then, because right now the padding, that compensates on margins, is on whole component, and anything besides Avatars hides under them in rtl:

![Screen Shot 2019-10-02 at 4 48 23 PM](https://user-images.githubusercontent.com/22763470/66049863-d4a2a600-e534-11e9-93df-13f1ea30f8b8.png)

So it's working in rtl, but it's not symmetrical css wise :) Which makes adding anything beside it wonky.

Reply:
🤔 ok I see now.
Bummer 😞 
👍 
If it won't work we might need to add the `rtl` state.